src/frontend/pages/Products/components/Filters.jsx

```diff
@@ -0,0 +1,224 @@
+          onChange={() =>
+            productDispatch({ type: "CATEGORY", payload: "Mouse" })
+          }
+          checked={productState.categories.includes("Mouse") ? true : false}
+        />
+        <label htmlFor="mouse">Mouse</label>
+      </div>
+      <div>
+        <input
+          type="checkbox"
+          name="headphones"
+          id="headphones"
+          onChange={() =>
+            productDispatch({ type: "CATEGORY", payload: "Headphones" })
+          }
+          checked={
+            productState.categories.includes("Headphones") ? true : false
+          }
+        />
+        <label htmlFor="headphones">Headphones</label>
+      </div>
+
+      <h3 className="pt-1 pb-1">Rating</h3>
+
+      <div>
+        <input
+          type="radio"
+          name="rating"
+          id="above4"
+          className="rating"
+          onChange={() => productDispatch({ type: "RATING", payload: 4 })}
+          checked={
+            productState.rating === ""
+              ? false
+              : productState.rating === 4
+              ? true
+              : false
+          }
+        />
+        <label htmlFor="above4">4 Stars and above</label>
+      </div>
+
+      <div>
+        <input
+          type="radio"
+          name="rating"
+          id="above3"
+          onChange={() => productDispatch({ type: "RATING", payload: 3 })}
+          checked={
+            productState.rating === ""
+              ? false
+              : productState.rating === 3
+              ? true
+              : false
+          }
+        />
+        <label htmlFor="above3">3 Stars and above</label>
+      </div>
+      <div>
+        <input
+          type="radio"
+          name="rating"
+          id="above2"
+          onChange={() => productDispatch({ type: "RATING", payload: 2 })}
+          checked={
+            productState.rating === ""
+              ? false
+              : productState.rating === 2
+              ? true
+              : false
+          }
+        />
+        <label htmlFor="above2">2 Stars and above</label>
+      </div>
+
+      <div>
+        <input
+          type="radio"
+          name="rating"
+          id="above1"
+          onChange={() => productDispatch({ type: "RATING", payload: 1 })}
+          checked={
+            productState.rating === ""
+              ? false
+              : productState.rating === 1
+              ? true
+              : false
+          }
+        />
+        <label htmlFor="above1">1 Star and above</label>
+      </div>
+
+      <h3 className="pt-1 pb-1">Sort By</h3>
+      <div>
+        <input
+          type="radio"
+          name="sort"
+          id="lowtohigh"
+          onChange={() =>
+            productDispatch({ type: "SORT", payload: "LOW_TO_HIGH" })
+          }
+          checked={
+            productState.sort === ""
+              ? false
+              : productState.sort === "LOW_TO_HIGH"
+              ? true
+              : false
+          }
```

Comment:
here also you can write  checked={
            productState.sort === "LOW_TO_HIGH"  }